style.css

```diff
@@ -0,0 +1,72 @@
+*{
+
+    margin: 0;
+    padding: 0;
+}
```

Comment:
It would be better to give space after selector name * { }. It shows that you are an experienced web designer.

Reply:
- [ ] Hi @abdullah97-pro 👋, Well done on using inline comments and giving a reason. 👍
- Kindly use some kind words so that it doesn't seem like an order but rather seems like a request. For example: `It would be better to manage your codes in the same line with indentation, as it is a very clear and good coding standard.`